TCMSApp/src/client/app/defects/defects.html

```diff
@@ -29,97 +32,90 @@
 
             </div>
             <hr>
-            <div class="well">
-                <div class="row">
-                    <div class="col-md-8">
-                        <h4>
-                            <strong>First Error</strong>
-                            <a class="fa fa-pencil" href=""></a>
-                            <a class="fa fa-trash" href=""></a>
-                        </h4>
-                    </div>
-                    <div class="col-md-4 text-right">
-                        <strong><span>Date:</span></strong>
-                        <span>11.11.2015</span>
-                        &nbsp;&nbsp;&nbsp;
-                        <strong><span>Priority:</span></strong>
-                        <span>High</span>
+
+            <div class="row">
+                <div class="col-sm-8">
+                    <div class="panel panel-info">
+                        <div class="panel-heading">
+                            Defects list
+                            <span class="pull-right">
+                                <span class="glyphicon glyphicon-inbox"></span>
+                                archive
+                            </span>
+                        </div>
+                        <table class="table">
+                            <thead>
+                            <tr>
+                                <th>
+                                    <input type="checkbox">
+                                </th>
+                                <th>
+                                    Name
+                                </th>
+                                <th>
+                                    Date
+                                </th>
+                                <th>
+                                    Priority
+                                </th>
+
+                            </tr>
+                            </thead>
+                            <tbody ng-repeat="tc in arrayDefects">
+                            <tr>
```

Comment:
don't forget to mark the line with the selected defect (current defect) somehow. 
try background color or bold font